modules/tensor_mechanics/src/materials/ComputeStressBase.C

```diff
@@ -41,7 +41,7 @@ ComputeStressBase::ComputeStressBase(const InputParameters & parameters) :
   const unsigned num = fcn_names.size();
 
   if (!(num == 0 || num == 3*3))
-    mooseError("Either zero or " << 3*3 << " initial stress functions must be provided to TensorMechanicsMaterial.  You supplied " << num << "\n");
+    mooseError("Either zero or ", 3*3, " initial stress functions must be provided to TensorMechanicsMaterial.  You supplied ", num, "\n");
```

Comment:
Strictly speaking there should be spaces around `*`, but it would have to be fixed in other places as well. So this fix is optional ;-)